<launch>
	<node name="rmdn_node" type="rmdn_pepper_node.py" pkg="rmdn_hri_ros" required="true" output="screen"/>
	<!-- <node pkg="rosbag" type="record" name="rosbag_record"
       args="record -o combined /tf /joint_states /display_robot_state /visualization_marker_array /usb_cam_node/image_raw"/>    -->


</launch>
